cockpit-frontend/src/components/TeamMemberList/TeamMemberList.jsx

```diff
@@ -1,62 +1,282 @@
-          ))}
-        </List>
-      )}
-      {mvpInfo.team && teamMembers.length === 0 && (
-        <div className={classes.noMembers}>No Team members yet</div>
-      )}
+                        <MenuItem value="" disabled>
+                          Select a Role
+                        </MenuItem>
+                        <MenuItem value="PO">PO</MenuItem>
+                        <MenuItem value="PPO">PPO</MenuItem>
+                        <MenuItem value="Scrum Master">Scrum Master</MenuItem>
+                        <MenuItem value="Technical Lead">
+                          Technical Lead
+                        </MenuItem>
+                        <MenuItem value="Application Developer">
+                          Application Developer
+                        </MenuItem>
+                        <MenuItem value="DevOps">DevOps</MenuItem>
+                      </Select>
+                    </FormControl>
+                  </Grid>
+                  <Grid item xs={12}>
+                    <FormLabel className={classes.formLabel}>Email</FormLabel>
+                    <TextField
+                      className={classes.textField}
+                      required
+                      fullWidth
+                      variant="outlined"
+                      placeholder="Member Email"
+                      size="small"
+                      type="email"
+                      onChange={(event) => setEmail(event.target.value)}
+                      value={email}
+                    />
+                  </Grid>
+                  {/*     <Grid item xs={12}>
+                  <FormLabel className={classes.formLabel}>
+                    Avatar URL
+                  </FormLabel>
+                  <TextField
+                    className={classes.textField}
+                    fullWidth
+                    variant="outlined"
+                    placeholder="http://"
+                    size="small"
+                    onChange={(event) => {
+                      setAvatarUrl(event.target.value);
+                    }}
+                    value={avatarUrl}
+                    error={!validUrl}
+                    helperText={!validUrl ? 'Incorrect image URL.' : ' '}
+                  />
+                  <img
+                    style={{ height: 0 }}
+                    alt=""
+                    src={avatarUrl}
+                    onLoad={() => setValidUrl(true)}
+                    onError={() => checkURL()}
+                  />
+                </Grid> */}
+                  <Grid item xs={5} />
+                  <Grid item xs={3}>
+                    <Button className={classes.addButton} onClick={closeForm}>
+                      Discard
+                    </Button>
+                  </Grid>
+                  <Grid item xs={1} />
+                  <Grid item xs={3}>
+                    <Button
+                      disabled={
+                        firstName.length === 0 ||
```

Comment:
What about create a function for this logic called field validator ?

Reply:
done